components/carousel/carousel.md

```diff
@@ -0,0 +1,36 @@
+Carousel:
+
+    <div>
+      <Carousel
+        images={[
+          'http://lorempixel.com/600/400/city',
```

Comment:
Not on this version, @EduardTrutsyk. There's a roadmap for a _generic carousel_. You'd input the children elements.